icu4c/source/i18n/decNumberLocal.h

```diff
@@ -166,7 +166,9 @@
 
   /* Set DECDPUNMAX -- the maximum integer that fits in DECDPUN       */
   /* digits, and D2UTABLE -- the initializer for the D2U table        */
-  #if   DECDPUN==1
+  #ifndef DECDPUN
```

Comment:
is this from upstream code?

Reply:
Possibly, but I'm not sure where I'd go about updating the upstream.  The [web site for decNumber](http://speleotrove.com/decimal/decnumber.html) points to [an old ICU link](http://download.icu-project.org/files/decNumber/) as its source of truth.